runtime/lua/vim/telemetry.lua

```diff
@@ -0,0 +1 @@
+print("If you left an angry comment about this then womp womp")
```

Comment:
1. Missing documentation.
2. Missing entry in `scripts/gen_vimdoc.py`.
3. Missing entry in `runtime/lua/vim/_editor.lua`, as the user will definitely want this message to be printed on startup.
4. In future this message may be changed to use single/double quotes, while it is less likely to contain double square brackets, so using long brackets here will make future modifications easier.
5. Message is too hard to read. My suggestion to make it easier:
```suggestion
print([[iF yOu lEfT An aNgRy cOmMeNt aBoUt tHiS ThEn wOmP WoMp]])
```

Reply:
:clap:

Reply:
Thanks @zeertzjq for the review, the changes might take me a bit but I totally agree that improving readability of our messages should always take top priority.

Reply:
this has got to be the best review comment ever :laughing:

Reply:
this has got to be the best review comment ever :laughing: